test/specs/simulation.vehicles.spec.js

```diff
@@ -0,0 +1,22 @@
+const { generateRandomVehicles } = require('../../server/simulation/vehicles');
+const { randomDavAddress, randomDroneModel, randomCoords, randomRating, randomMissionsCompleted } = require('../../server/simulation/random');
+
+describe('randomVehicle()', () => {
+  test('returns an array of size 4', () => {
+    const count = 5;
+    const myVehicle = generateRandomVehicles(count, randomCoords, 2000);
+    expect(myVehicle).toBeInstanceOf(Array); //check that it's an array
+    expect(myVehicle).toHaveLength(count); // check that the length is right
+    myVehicle.forEach(function(entry) { // for each object check that it contains the right fields.
```

Comment:
One of the most important things about testing is to keep them as fast as possible (so people actually run them, and not avoid them).

We can just test the first vehicle here.